qdax/tasks/hypervolume_functions.py

```diff
@@ -0,0 +1,94 @@
+from typing import Callable, Tuple
+
+import jax
+import jax.numpy as jnp
+
+from qdax.types import Descriptor, ExtraScores, Fitness, Genotype, RNGKey
+
+
+def square(params: Genotype) -> Tuple[Fitness, Descriptor]:
+    """
+    Seach space should be [0,1]^n
+    BD space should be [0,1]^n
+    GETTING THE EXPECTED RESULT BUT DIFFERENT FROM PAPER FUNCTION
```

Comment:
Which paper are you talking about?

Reply:
Is it the one from Stephane Doncieux? Could you add a link in the dosctrings?

Reply:
It is the one from Jean-Baptiste Mouret, I don't think the paper is online yet.